src/liquid/components/ld-input/ld-input.tsx

```diff
@@ -357,8 +357,8 @@ export class LdInput implements InnerFocusable, ClonesAttributes {
   }
 
   disconnectedCallback() {
-    /* istanbul ignore next */
-    this.attributesObserver?.disconnect()
+    /* istanbul ignore if */
+    if (this.attributesObserver) this.attributesObserver.disconnect()
```

Comment:
This is explicitly telling that we intentionally do not test this branch (because it's not worth it). This is better than seeing missing coverage and asking yourself over and over again whether we need to write tests. I'd like to keep it like this.